README.md

```diff
@@ -6,47 +6,30 @@ direnv -- unclutter your .profile
 [![latest packaged version(s)](https://repology.org/badge/latest-versions/direnv.svg)](https://repology.org/project/direnv/versions)
 [![Support room on Matrix](https://img.shields.io/matrix/direnv:numtide.com.svg?label=%23direnv%3Anumtide.com&logo=matrix&server_fqdn=matrix.numtide.com)](https://matrix.to/#/#direnv:numtide.com)
 
-`direnv` is an extension for your shell. It augments existing shells with a
-new feature that can load and unload environment variables depending on the
-current directory.
+`direnv` is an extension for your unix shell. It augments existing shells with a
```

Comment:
Maybe "terminal shell"? Direnv also works with PowerShell.

Reply:
I think the original was alright. Just thought most users were on Unix.

Reply:
I tried to get it working in PowerShell (or cmd.exe) but I couldn't get it working.